sites/fast-website/src/public/footer.ejs

```diff
@@ -5,7 +5,7 @@
         <site-navigation-item href="https://go.microsoft.com/fwlink/?LinkID=206977">Terms of use</site-navigation-item>
         <site-navigation-item href="https://www.microsoft.com/trademarks">Trademarks</site-navigation-item>
         <site-navigation-item>
-            <p>&copy; Microsoft 2021</p>
+            <p>&copy; Microsoft ${new Date().getFullYear()}</p>
```

Comment:
I was looking at this and didn't remember this being valid ejs. Does this work?

Reply:
Oops, you're right @EisenbergEffect , thank you!